src/WebdriverClassicDriver.php

```diff
@@ -1227,5 +1240,21 @@ private function getElementDomProperty(RemoteWebElement $element, string $proper
         }
     }
 
+    /**
+     * @param mixed $value
+     * @throws DriverException
+     */
+    private function getAsString($value, string $name): string
```

Comment:
Why this method is needed? Previously we trusted JavaScript to return the correct value, but now we don't.

Reply:
PHPStan (correctly) warns that we may get values that are not stringable (there's a test for that too), in which case all sorts of weird things can happen (in some cases a type error is thrown, in other cases we break the contract with the library user and provide the wrong type).

In any case, there are 4-5 cases where that method is useful, so it makes sense to avoid repeating the same kind of check.

Reply:
I understand now. Thank you.